randovania/games/fusion/logic_database/header.txt

```diff
@@ -76,6 +76,17 @@ Templates
           Can Use Power Bombs
           Screw Attack and Knowledge (Beginner)
 
+* Can Bounce in Ball:
+      Can Use Bombs or Can Use Springball
+
+* Can Screw Single Bomb Blocks:
+      Trivial
```

Comment:
is this supposed to be trivbial?

Reply:
oops, ignore that. I spelled the template wrong and made a new one. I already deleted it in the json.